cluster/gce/util.sh

```diff
@@ -2551,7 +2567,7 @@ function delete-all-firewall-rules() {
 
 # Ignores firewall rule arguments that do not exist in NETWORK_PROJECT.
 function delete-firewall-rules() {
-  for fw in $@; do
+  for fw in "$@"; do
```

Comment:
I guess since the firewall rules will probably never contain things like whitespaces, we will never see a bug here. But if they did, the current code would split such an argument.